manticore/ethereum.py

```diff
@@ -2586,10 +2587,10 @@ def flagged(flag):
                 tx_summary.write("Value: %d %s\n" % (state.solve_one(tx.value), flagged(issymbolic(tx.value))))
                 tx_summary.write("Gas used: %d %s\n" % (state.solve_one(tx.gas), flagged(issymbolic(tx.gas))))
                 tx_data = state.solve_one(tx.data)
-                tx_summary.write("Data: %s %s\n" % (binascii.hexlify(tx_data), flagged(issymbolic(tx.data))))
+                tx_summary.write("Data: {0}{1} {2}\n".format('0x', binascii.hexlify(tx_data).decode('utf-8'), flagged(issymbolic(tx.data))))
```

Comment:
The `'0x'` can be moved into docstring.

Reply:
This should be addressed by commit [3b16a06](https://github.com/trailofbits/manticore/pull/1117/commits/3b16a0613ecc3d925810f654fa28aca81824055f).